src/metabase/models/interface.clj

```diff
@@ -38,6 +39,10 @@
 
 (set! *warn-on-reflection* true)
 
+;;; even tho this gets loaded by `.init`, it's important for REPL usage it gets loaded ASAP so other model namespaces
```

Comment:
unrelated but I noticed when I loaded `metabase.warehouses.models.database` from the REPL `(t2/table-name :model/Field)` didn't work correctly until I loaded `metabase.models.resolution`. During normal usage the `init` namespace would load it but I want it to be REPL-friendly as well so I'm adding an extra `require` here